versioned/persist/rocks/src/main/java/org/projectnessie/versioned/persist/rocks/RocksDatabaseAdapter.java

```diff
@@ -598,7 +598,7 @@ protected void doRemoveFromNamedReferences(
         throw new RuntimeException(e);
       }
 
-      ReferenceNames.Builder newRefNames = referenceNames.toBuilder();
+      ReferenceNames.Builder newRefNames = referenceNames.toBuilder().clearRefNames();
```

Comment:
whats the value of clearing the builder we get from `referenceNames` instead of just using a new one?